spec/cypress/integration/ChallengePartnership.feature

```diff
@@ -27,20 +27,21 @@ Feature: Reporting an error with a partnership
 
   Scenario: A logged in induction tutor for FIP school challenges partnership
     Given I am logged in as existing user with email "test-subject@example.com"
-    And I am on "2021 school partnerships" page
+    And I am on "2021 school partnerships" page with id "0000bd75-31d0-4eb3-8df3-07f866e41d51"
     When I click on "link" containing "report that your school has been confirmed incorrectly"
     Then I should be on "challenge partnership (any token)" page
 
     When I click on "I do not recognise this training provider" label
     And I click the submit button
     Then I should be on "challenge partnership success" page
 
-    When I navigate to "2021 school partnerships" page
+    When I navigate to "2021 school partnerships" page with id "0000bd75-31d0-4eb3-8df3-07f866e41d51"
     Then "page body" should contain "Signing up with a training provider"
 
+    @focus
   Scenario: A logged in induction tutor for CIP school challenges partnership
     Given I am logged in as existing user with email "test-subject2@example.com"
-    And I am on "2021 school cohorts" page
+    And I am on "2021 school cohorts" page with id "00041221-d612-46a8-a096-87ad63ff3a7d"
```

Comment:
Hmm, could we use pretty url slugs here instead of ids?

Reply:
I guess Callum was adding that change, and you will have a lot of conflicts with him 😂